content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,75 @@
+
+## Traffic Management
+
+- **Added** the capability for Istio to automatically determine protocol types avoiding the need for specifying the mandatory container ports field in the pod spec.
+- **Added** a mode to the Gateway API for mutual TLS operation that is capable of reading certificates via SDS.
+- **Added** enhancements to the `EnvoyFiltier` API such that all configuration can be added or updated.
+- **Fixed** Sidecar proxy readiness checks now check Envoy's readiness status.
+- **Fixed** issues with protocols like MySQL and MongoDB where the service communicates over the network first in permissive mutual TLS mode.
+- **Improved** Redis load balancer defaults to `MAGLEV` when using the Redis proxy.
+- **Improved** traffic management to direct traffic to the same region and zone.
+- **Improved** Pilot performance.  CPU utilization may be up to 10 times lower.
+- **Improved** the capabilities of `ServiceEntries` to enable the same hostname in different namespaces.
+- **Improved** the capability of the Sidecar API to customize `OutboundTrafficPolicy`.
+
+## Security
+
+- **Added** a unified logging pattern to Citadel agent.
+- **Added** metrics into Citadel agent to monitor SDS push.
+- **Added** the capability to ingress gateway to provide client CA certificates when using SDS.
+- **Added** the capability for trust domain validation when sidecars use mutual TLS.
+- **Added** the labels `ca.istio.io/env` and `ca.istio.io/override`  to allow operators to control service account secret generation by namespace.
+- **Removed** integration with Vault CA temporarily. This was caused by SDS requirements and will be reintroduced in a new release.
+- **Improved** the performance of secrets handling in the sidecars by introducing SDS.
+- **Improved** security and reliability by enabling the the Envoy JWT filter by default.
+- **Improved** the reliability of SDS by requiring Kubernetes 1.13. Please be aware that while Istio supports Kubernetes 1.12, SDS [requires Kubernetes 1.13](/blog/2019/trustworthy-jwt-sds).
```

Comment:
Istio *works* on 1.12, but we only *support* 1.13-1.15 by our official policy

Reply:
I wasn't able to get SDS working for 1.3rc running on k8s 1.12 (could be anecdotal)

Reply:
SDS does not work with 1.12, but if you don't use SDS it *should*. But Istio's official policy is 3 kubernetes releases (1.13-1.15)

Reply:
adjusted.